javascripts/search.js

```diff
@@ -201,12 +201,12 @@ search.filter('description', function($sce) {
   }
 });
 
-search.filter('question', function($sce) {
+search.filter('question', function() {
   return function(result) {
     if (result.highlight && result.highlight._source.sys_content_plaintext) {
-      return $sce.trustAsHtml(result.highlight._source.sys_content_plaintext[0]);
+      return String(result.highlight._source.sys_content_plaintext[0]).replace(/&lt;/g,"<").replace(/&gt;/g,">");
     }
-    return $sce.trustAsHtml(result._source.sys_content_plaintext);
+    return String(result._source.sys_content_plaintext).replace(/&lt;/g,"<").replace(/&gt;/g,">");
```

Comment:
The best solution appears to be something similar, but containing the replacement for both open and close tags:
`return text ? String(text).replace(/<[^>]+>/gm, '') : '';`